nni/common/concrete_trace_utils/passes/kwargs_shape_prop.py

```diff
@@ -0,0 +1,93 @@
+import torch
+import traceback
+from torch.fx.node import Node, map_aggregate
+from typing import Optional, Union, NamedTuple, Tuple, Any, Dict
+from ..kwargs_interpreter import KwargsInterpreter
+
+class TensorMetadata(NamedTuple):
```

Comment:
Can reuse the original one from shape prop actually. I believe they are the same.

Reply:
Sure. Updated in the latest commit.